onnx/version_converter/adapters/axis_attribute_to_input.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) ONNX Project Contributors
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE {
+namespace version_conversion {
+
+class AxisAttributeToInput : public Adapter {
+ public:
+  explicit AxisAttributeToInput(const std::string& op_name, const OpSetID& initial, const OpSetID& target)
+      : Adapter(op_name, initial, target) {}
+
+  Node* adapt(std::shared_ptr<Graph> graph, Node* node) const override {
+    if (node->hasAttribute(kaxis)) {
```

Comment:
I guess this will be changed to handle a change in the default-value?

Reply:
👍 that's the plan

Reply:
Moved to a separate PR